### DISCUSSION TITLE
Print head stops momentarily with all firmares after version 1.3.3 without HS

### DISCUSSION THREAD

**Discussion by manpres2** [Jyers/Marlin#1432] [Q&A] [answered]
Hi,

I already have a previous thread but that was messed up with unhelpful advice's. 

Problem: print head stops momentarily for around solid 2 seconds and then starts moving, i noticed its dong that during retractions. When this happens it leaves a  indent in the print surface.

Troubleshooting steps tried: 

1. disabled power loss recovery nogo 
2. changed memory card nogo
3. changed firmware versions starting from 1.3.4 to 1.3.5 HS and non HS nogo
4. it works fine on version 1.3.3 non HS .
HS means high speed probe where BL touch just touches the bed but do not clock back.

It some how was working fine when i was using Octoprint, cant use RPi anymore as i am using klipper on it with ender 5 plus.

Is there any logs i can pull for this ?

**Comment by Jyers** [comment, marked as answer]
Perhaps try this with the latest source and see if it helps. A new release will be out soon
